src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -270,7 +302,29 @@ Please note that the `<index>` denotes the number/index (in the form of a
 nonnegative integer) of the echo not the echo time value which needs to be stored in the
 field EchoTime of the separate JSON file.
 
-Some meta information about the acquisition MUST be provided in an additional
+Complex-valued data MUST be split into one file for each data type.
+For BOLD data, there are separate suffixes for magnitude (`_bold`) and phase
+(`_phase`) data, but the `_phase` suffix is considered a legacy suffix
+(i.e., still supported in the specification, but to be deprecated later).
```

Comment:
good place to insert a link about our definition of deprecation here, see https://github.com/bids-standard/bids-specification/issues/611

Reply:
Agreed, although I was planning to wait until a corresponding PR was merged.

Reply:
@tsalo #634 is merged. Want to update this? This PR overall LGTM, and this is the only thing left to change.

Reply:
Good catch. I've updated the text. It would be nice to link directly to the deprecation definition, but I did the next best thing and links to the Definitions section.